d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadLoadBalancer.java

```diff
@@ -53,22 +55,36 @@
  * In DUAL_READ mode, it reads from both the old and the new load balancer, but relies on the data from old
  * load balancer only.
  */
-public class DualReadLoadBalancer implements LoadBalancerWithFacilities
-{
+public class DualReadLoadBalancer implements LoadBalancerWithFacilities {
```

Comment:
Ditto on the brace formatting, you have to leave it on the new line

Reply:
Updated